test/max_bgw_8.conf.in

```diff
@@ -0,0 +1,21 @@
+# NOTE: any changes here require changes to two other places
+# 1) tsl/test/postgresql.conf requires that it's prefix be the same as this file.
+# 2) appveyor.yml needs to be updated with the same options
+
+shared_preload_libraries=timescaledb
+max_worker_processes=16
```

Comment:
Maybe make this 24, because the default `ts_guc_max_background_workers` is 16 already, at least for `tsl/test/postgresql.conf.in`?